deepmd/dpmodel/train/entrypoint.py

```
@@ -0,0 +1,167 @@
+    input_file: str
+    output: str = "out.json"
+    init_model: str | None = None
+    restart: str | None = None
+    init_frz_model: str | None = None
+    finetune: str | None = None
+    model_branch: str = ""
+    use_pretrain_script: bool = False
+    skip_neighbor_stat: bool = False
+
+
+class AbstractTrainEntrypoint(ABC):
```

Ruff (0.15.18)

[warning] `AbstractTrainEntrypoint` is an abstract base class, but it has no abstract methods or properties

(B024)